CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# NEXT RELEASE
+# 13.26.0 Release notes
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
```

Comment:
should this be removed? Same for `Fixed` section.

Reply:
I changed the file and did not commit it  💯